lib/solr_wrapper.rb

```diff
@@ -10,6 +10,10 @@ def self.default_instance_options
     @default_instance_options ||= {}
   end
 
+  def self.default_instance_options=(options)
```

Comment:
@cbeer I will give it a go, but I was not sure based on the fact that these were class methods.